<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DegradedUI</class>
 <widget class="QWidget" name="DegradedUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>310</width>
    <height>289</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>10</number>
   </property>
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="rightMargin">
    <number>10</number>
   </property>
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <item>
    <widget class="QFrame" name="frameDegradedSettings">
     <property name="styleSheet">
      <string notr="true">
/*
QFrame {

    border: 1px solid rgb(0,106,255);
    border-radius: 3px;
}

*/</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <item>
       <widget class="QFrame" name="frameColor1">
        <property name="styleSheet">
         <string notr="true">

QFrame {

    border: 0px solid rgb(0,106,255);
    border-radius: 3px;
}

</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>3</number>
         </property>
         <property name="leftMargin">
          <number>3</number>
         </property>
         <property name="topMargin">
          <number>3</number>
         </property>
         <property name="rightMargin">
          <number>3</number>
         </property>
         <property name="bottomMargin">
          <number>3</number>
         </property>
         <item alignment="Qt::AlignHCenter">
          <widget class="QToolButton" name="tbColor1">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Choix de la couleur 1</string>
           </property>
           <property name="styleSheet">
            <string notr="true"> QToolButton:pressed {
 background-color: white;
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #f6f7fa, stop: 1 #dadbde);

 }
 


QToolButton:hover:!pressed
{
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #dadbde, stop: 1 #f6f7fa);
}QToolButton {

    border: 1px solid rgb(0,106,255);
    border-radius: 3px;
}

</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="iconSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignHCenter">
          <widget class="QLabel" name="label">
           <property name="styleSheet">
            <string notr="true">

QWidget {
  
	color : rgb(0,106,255);
     font:  9pt Calibri;


 }


QWidget:!enabled {
  
	color : rgb(150,150,150);
     font:  9pt Calibri;
 }
</string>
           </property>
           <property name="text">
            <string>Couleur 1</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frameColor2">
        <property name="styleSheet">
         <string notr="true">

QFrame {

    border: 0px solid rgb(0,106,255);
    border-radius: 3px;
}

</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>3</number>
         </property>
         <property name="leftMargin">
          <number>3</number>
         </property>
         <property name="topMargin">
          <number>3</number>
         </property>
         <property name="rightMargin">
          <number>3</number>
         </property>
         <property name="bottomMargin">
          <number>3</number>
         </property>
         <item alignment="Qt::AlignHCenter">
          <widget class="QToolButton" name="tbColor2">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Choix de la couleur 2</string>
           </property>
           <property name="styleSheet">
            <string notr="true"> QToolButton:pressed {
 background-color: white;
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #f6f7fa, stop: 1 #dadbde);

 }
 


QToolButton:hover:!pressed
{
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #dadbde, stop: 1 #f6f7fa);
}QToolButton {

    border: 1px solid rgb(0,106,255);
    border-radius: 3px;
}

</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="iconSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignHCenter">
          <widget class="QLabel" name="label_2">
           <property name="styleSheet">
            <string notr="true">

QWidget {
  
	color : rgb(0,106,255);
     font:  9pt Calibri;


 }


QWidget:!enabled {
  
	color : rgb(150,150,150);
     font:  9pt Calibri;
 }
</string>
           </property>
           <property name="text">
            <string>Couleur 2</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frameDegradedType">
        <property name="styleSheet">
         <string notr="true">

QFrame {

    border: 0px solid rgb(0,106,255);
    border-radius: 3px;
}

</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item alignment="Qt::AlignHCenter">
          <widget class="QComboBox" name="cbDegradedType">
           <property name="minimumSize">
            <size>
             <width>51</width>
             <height>75</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>51</width>
             <height>75</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Choix du type de dégradé</string>
           </property>
           <property name="styleSheet">
            <string notr="true">QComboBox {
    border: 1px solid rgb(0,106,255);
    border-radius: 3px;
  padding: -0px -0px -0px 0px;

	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #dadbde, stop: 1 #f6f7fa);
}

QComboBox::drop-down {
	width: 0px;
}

QComboBox:hover{
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #f6f7fa, stop: 1 #dadbde);
}


 QComboBox QAbstractItemView {
     selection-background-color: lightgray;
 }</string>
           </property>
           <property name="currentIndex">
            <number>0</number>
           </property>
           <property name="iconSize">
            <size>
             <width>51</width>
             <height>75</height>
            </size>
           </property>
           <property name="frame">
            <bool>true</bool>
           </property>
           <item>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../resources.qrc">
              <normaloff>:/images/degraded/padSpread</normaloff>:/images/degraded/padSpread</iconset>
            </property>
           </item>
           <item>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../resources.qrc">
              <normaloff>:/images/degraded/radReflectSpread</normaloff>:/images/degraded/radReflectSpread</iconset>
            </property>
           </item>
           <item>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../resources.qrc">
              <normaloff>:/images/degraded/conical</normaloff>:/images/degraded/conical</iconset>
            </property>
           </item>
          </widget>
         </item>
         <item alignment="Qt::AlignHCenter">
          <widget class="QLabel" name="laDegraded">
           <property name="styleSheet">
            <string notr="true">

QWidget {
  
	color : rgb(0,106,255);
     font:  9pt Calibri;


 }


QWidget:!enabled {
  
	color : rgb(150,150,150);
     font:  9pt Calibri;
 }</string>
           </property>
           <property name="text">
            <string>Type de dégradé</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="maximumSize">
      <size>
       <width>1</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">	background-color: rgb(0,106,255);</string>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frameDrawZone">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="vlDrawZone">
      <property name="spacing">
       <number>3</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <item alignment="Qt::AlignHCenter">
       <widget class="QFrame" name="frame">
        <property name="styleSheet">
         <string notr="true">
/*
QFrame {

    border: 1px solid rgb(0,106,255);
    border-radius: 3px;
}

*/</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <property name="leftMargin">
          <number>5</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>5</number>
         </property>
         <property name="bottomMargin">
          <number>5</number>
         </property>
         <property name="horizontalSpacing">
          <number>3</number>
         </property>
         <item row="1" column="0">
          <widget class="QRadioButton" name="rbColor2">
           <property name="toolTip">
            <string>Utiliser la couleur 2 comme fond</string>
           </property>
           <property name="styleSheet">
            <string notr="true">

QWidget {
  
	color : rgb(0,106,255);
     font:  9pt Calibri;


 }


QWidget:!enabled {
  
	color : rgb(150,150,150);
     font:  9pt Calibri;
 }
</string>
           </property>
           <property name="text">
            <string>Utiliser couleur 2</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QRadioButton" name="rbDegraded">
           <property name="toolTip">
            <string>Utiliser un dégradé de la couleur 1 et 2 pour le fond</string>
           </property>
           <property name="styleSheet">
            <string notr="true">

QWidget {
  
	color : rgb(0,106,255);
     font:  9pt Calibri;


 }


QWidget:!enabled {
  
	color : rgb(150,150,150);
     font:  9pt Calibri;
 }
</string>
           </property>
           <property name="text">
            <string>Utiliser dégradé</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QRadioButton" name="rbColor1">
           <property name="toolTip">
            <string>Utiliser la couleur 1 comme fond</string>
           </property>
           <property name="styleSheet">
            <string notr="true">

QWidget {
  
	color : rgb(0,106,255);
     font:  9pt Calibri;


 }


QWidget:!enabled {
  
	color : rgb(150,150,150);
     font:  9pt Calibri;
 }
</string>
           </property>
           <property name="text">
            <string>Utiliser couleur 1</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
